packages/dsh-tauri/src/client/compat.ts

```
@@ -0,0 +1,151 @@
+  return typeof fn === 'function' ? (workspaceId?: string) => fn.call(owner, workspaceId) : undefined
+}
+
+/** Adapt alpha's nested list/navigation services to the rc.2 plugin contract. */
+export function compat(ctx: ClientContext): ClientContext {
+  const safeLookup = (name: string): unknown => lookup(ctx, name)
+
+  // Inline Alpha Detection: 判断当前环境是否为 Alpha 版本
+  const alphaSessions = safeLookup('sessions') as RuntimeObject | undefined
+  const isAlpha
+    = (alphaSessions?.list !== undefined && typeof alphaSessions.getSnapshot !== 'function')
+    || safeLookup('uiWorkspace') !== undefined
```

ESLint

[error] Expected indentation of 6 spaces

(style/indent-binary-ops)